src/main/java/core/basesyntax/service/FruitOperationGenerator.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.entity.Fruit;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.entity.FruitTransaction;
+import core.basesyntax.converter.ConverterCSVToObject;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+public class FruitOperationGenerator {
+
+    public void fruitRestReport() throws FileNotFoundException {
+        String inputFile = "src/main/resources/fruitShop.csv";
+        ConverterCSVToObject converterCSVToObject = new ConverterCSVToObject();
+        List<FruitTransaction> fruitTransactionList = converterCSVToObject.reader(inputFile);
```

Comment:
you have already read it in Main, call forEach in Main and just pass each fruit here as argument